chart/istio-pod-network-controller/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Default values for istio-pod-network-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+# this property can have two values, Kubernetes or OpenShift
+kubernetesDistribution: Kubernetes
+Namespace: istio-system
+
+image:
+  repository: quay.io/raffaelespazzoli/istio-pod-network-controller
```

Comment:
Are we going to move toward the location in quay that automatically builds when changes are made to this repository?